### PR TITLE
Add Apple talks

### PR DESCRIPTION
**Pull request by aokolnychyi**
This PR adds Apple talks given over the years.

### CODE REVIEW

**Comment by aokolnychyi** [comment]
@nastra @Fokko @rdblue @szehon-ho @RussellSpitzer @danielcweeks

File: landing-page/content/common/talks.md [hunk 1]
```diff
@@ -24,12 +24,33 @@ disableSidebar: true
...
+**Date**: January 12, 2023, **Author**: Szehon Ho
+
 ### [Data architecture in 2022](https://www.youtube.com/watch?v=1oXmBbB77ak)
 **Date**: May 5, 2022, **Authors**: Ryan Blue
 
 ### [Why You Shouldn’t Care About Iceberg | Tabular](https://www.youtube.com/watch?v=_GW3GYZK66U)
 **Date**: March 24, 2022, **Authors**: Ryan Blue
 
+### [Managing Data Files in Apache Iceberg](https://www.dremio.com/resources/webinars/managing-data-files-in-apache-iceberg/)
+**Date**: Marc 2, 2022, **Author**: Russell Spitzer
```
Comment (Fokko) [inline landing-page/content/common/talks.md:46]:
```suggestion
**Date**: March 2, 2022, **Author**: Russell Spitzer
```

File: landing-page/content/common/talks.md [hunk 2]
```diff
@@ -24,12 +24,33 @@ disableSidebar: true
...
+
+### [Tuning Row-Level Operations in Apache Iceberg](https://www.dremio.com/resources/webinars/tuning-row-level-operations-in-apache-iceberg/)
+**Date**: Marc 2, 2022, **Author**: Anton Okolnychyi
```
Comment (Fokko) [inline landing-page/content/common/talks.md:49]:
```suggestion
**Date**: March 2, 2022, **Author**: Anton Okolnychyi
```

**Review by Fokko — APPROVED** [review]
LGTM, thanks @aokolnychyi

**Comment by aokolnychyi** [comment]
Thanks, @Fokko!

**Pull request merged**